packages/@coorpacademy-components/src/template/mobile-login/welcome/index.native.tsx

```diff
@@ -0,0 +1,219 @@
+    gradient: {
+      position: 'absolute',
+      top: -100,
+      bottom: 0,
+      left: -180,
+      right: 0,
+      opacity: 1,
+      transform: [{rotate: '35deg'}]
+    },
+    gradient2: {
+      position: 'absolute',
+      top: -80,
+      bottom: 0,
+      left: 0,
+      right: -200,
+      opacity: 0.5,
+      transform: [{rotate: '-35deg'}]
+    },
+    logo: {
+      padding: 100
+    },
+    title: {
+      fontWeight: '600',
+      fontSize: 28,
+      lineHeight: 36,
+      paddingVertical: 8
+    },
+    description: {
+      fontSize: 18,
+      lineHeight: 24,
+      paddingVertical: 8
+    },
+    actions: {
+      width: '100%'
+    },
+    buttons: {
+      paddingVertical: 20,
+      alignItems: 'center'
+    },
+    button: {
+      paddingVertical: 12,
+      marginVertical: 4,
+      width: '100%',
+      flexDirection: 'row',
+      justifyContent: 'center',
+      borderRadius: 12,
+      alignItems: 'center',
+      overflow: 'hidden'
+    },
+    ctaQRCode: {
+      backgroundColor: theme.colors.cta
+    },
+    ctaQRCodeText: {
+      marginLeft: 8,
+      color: '#fff',
+      fontWeight: '700',
+      fontSize: 14,
+      lineHeight: 24
+    },
+    qrCodeIcon: {
+      fill: '#fff',
+      height: 14,
+      width: 14
+    },
+    ctaReceiveMail: {
+      backgroundColor: '#eaeaeb'
+    },
+    ctaReceiveMailText: {
+      marginLeft: 8,
+      color: '#1D1D2B',
+      fontWeight: '700',
+      fontSize: 14,
+      lineHeight: 24
+    },
+    mailIcon: {
+      fill: '#1D1D2B',
+      height: 12,
+      width: 16
+    },
+    help: {
+      flexDirection: 'row',
+      justifyContent: 'center'
+    },
+    titleHelp: {
+      color: '#9999A8'
+    },
+    ctaHelp: {
+      marginLeft: 5,
+      textDecorationLine: 'underline',
+      color: '#9999A8'
+    }
+  });
+
+export type Props = {
+  onDemoPress: () => void;
+  onDesktopButtonPress: () => void;
+  onHelpPress: () => void;
+  onMobileButtonPress: () => void;
+  locales: {
+    title: string;
+    description: string;
+    ctaQRCode: string;
+    ctaReceiveMail: string;
+    titleHelp: string;
+    ctaHelp: string;
+  };
+};
+
+const Welcome = (props: Props) => {
+  const {locales, onDemoPress, onDesktopButtonPress, onHelpPress, onMobileButtonPress} = props;
+  const {theme} = useTemplateContext();
+  const [styleSheet, setStylesheet] = useState<StyleSheetType | null>(null);
+
+  useEffect(() => {
+    const _stylesheet = createStyleSheet(theme);
+    setStylesheet(_stylesheet);
+  }, [theme]);
+
+  if (!styleSheet) {
+    return null;
+  }
+
+  return (
+    <View style={styleSheet.wrapper} testID="welcome">
+      <LinearGradient
+        colors={['#0061FF', '#fff']}
+        locations={[0, 0.3]}
+        style={styleSheet.gradient}
+      />
+      <LinearGradient
+        colors={['#2199AB', '#fff']}
+        locations={[0, 0.3]}
+        style={styleSheet.gradient2}
+      />
+      <Touchable onLongPress={onDemoPress} style={styleSheet.logo}>
```

Comment:
what will be the use of "onDemoPress" ?

Reply:
https://go1.slack.com/archives/C04QB2T6T7D/p1680601980562879?thread_ts=1680598136.410269&cid=C04QB2T6T7D